packages/core/data-transfer/src/strapi/providers/remote-source/index.ts

```diff
@@ -210,7 +212,8 @@ class RemoteStrapiSourceProvider implements ISourceProvider {
     }
 
     this.ws = ws;
-    this.dispatcher = createDispatcher(this.ws);
+    const { retryMessageMaxRetries, retryMessageTimeout } = this.options;
```

Comment:
Should we check the numbers? that they are positive and reasonable? wdyt?

Reply:
yeah maybe for the time out, if the max retries is 0 or under it simply wouldn't retry which is fine but for the timeout I can check what the default behavior is if it's negative

Reply:
it would just start sending messages very quickly which is fine it will just throw and error after the retries, that would be there fault at this point